ibis/expr/tests/test_interactive.py

```diff
@@ -30,6 +30,12 @@ def test_interactive_execute_on_repr(self):
 
         assert len(self.con.executed_queries) > 0
 
+    def test_png_repr_returns_correct_type(self):
+        table = self.con.table('functional_alltypes')
+
+        with config.option_context('interactive', True):
```

Comment:
maybe would be good/useful to add here also a test for interactive False?

Reply:
Thought on that, it's a good point. But that's trickier, as it depends on other factors like libraries that are installed. I think it's out of scope here.

Reply:
ok sounds good to me!